Detectors/TRD/simulation/src/Trap2CRU.cxx

```diff
@@ -682,149 +556,102 @@ void Trap2CRU::convertTrapData(o2::trd::TriggerRecord const& triggerrecord, cons
     int side = cru % 2; // first cru is A second is B, 3rd is A etc
     mFeeID = constructTRDFeeID(supermodule, side, mEndPointID);
     mCruID = halfcru / 2;
-    mLinkID = o2::trd::constants::TRDLINKID;
     mEndPointID = halfcru % 2; // just the upper or lower half of the cru, hence %2 of the the halfcru number.
-    std::string sside;
-    (side) ? sside = "C" : sside = "A";
-    //15 links per half cru or cru end point.
-    memset(&mRawData[0], 0, sizeof(mRawData[0]) * mRawData.size()); //   zero the rawdata storage
+    // 15 links per half cru or cru end point.
     HalfCRUHeader halfcruheader;
     //now write the cruheader at the head of all the data for this halfcru.
-    buildHalfCRUHeader(halfcruheader, triggerrecord.getBCData().bc, halfcru);
+    buildHalfCRUHeader(halfcruheader, ir.bc, halfcru);
```

Comment:
@bazinski here I am using the shifted trigger time

Reply:
ah cool, I could not find it ;-)